cf-servicesinfo.rb

```diff
@@ -3,107 +3,96 @@
+    while more_spaces
+      spaces = Helpers.cfmunge("/v2/organizations/#{org['metadata']['guid']}/spaces?page=#{page}&results-per-page=100")
+      if spaces.length > 0
+        spaces.each do |space|
+          next if exempt_space(space['entity']['name'])
+          all_spaces[space['metadata']['guid']] = space['entity']['name']
+        end
+      else
+        more_spaces = false
+      end
+      page += 1
+    end
+  end
 
-	all_spaces = Hash.new
-
-	cfmunge('/v2/organizations?results-per-page=100').each do |org|
-		next if org['entity']['name'].include? "sandbox"
-
-		page = 1
-		more_spaces = true
-		while more_spaces
-			spaces = cfmunge("/v2/organizations/#{org['metadata']['guid']}/spaces?page=#{page}&results-per-page=100")
-			if spaces.length > 0
-				spaces.each do |space|
-					next if exempt_space(space['entity']['name'])
-					all_spaces[space['metadata']['guid']] = space['entity']['name']
-				end
-			else
-				more_spaces = false
-			end
-			page += 1
-		end
-	end
-
-	puts "All spaces #{all_spaces.length}"
-
-	all_spaces
+  puts "All spaces #{all_spaces.length}"
 
+  all_spaces
 end
 
 
 def get_all_services
+  services = {}
+  Helpers.cfmunge('/v2/services?results-per-page=100').each do |service|
+    services[service['metadata']['guid']] = service['entity']['description']
+  end
 
-	services = Hash.new
-	cfmunge('/v2/services?results-per-page=100').each do |service|
-		services[service['metadata']['guid']] = service['entity']['description']
-	end
-
-	services
+  services
 end
 
 def get_used_service_plans(services, spaces)
+  service_plans = {}
+
+  Helpers.cfmunge('/v2/service_plans?results-per-page=100').each do |service_plan|
+    service_plans[service_plan['metadata']['guid']] = {
+      'service_guid' => service_plan['entity']['service_guid'],
+      'expanded_name' => services[service_plan['entity']['service_guid']] + ' : ' + service_plan['entity']['name'],
+      'service_instance_count' => 0
+    }
+  end
 
-	service_plans = Hash.new
-
-	cfmunge('/v2/service_plans?results-per-page=100').each do |service_plan|
-		service_plans[service_plan['metadata']['guid']] = {
-			'service_guid' => service_plan['entity']['service_guid'],
-			'expanded_name' => services[service_plan['entity']['service_guid']] + ' : ' + service_plan['entity']['name'],
-			'service_instance_count' => 0
-		}
-	end
-
-	page = 1
-	while page < 3
-		cfmunge("/v2/service_instances?page=#{page}&results-per-page=100").each do |service_instance|
-			if spaces[service_instance['entity']['space_guid']]
-				service_plans[service_instance['entity']['service_plan_guid']]['service_instance_count'] += 1
-			end
-		end
-		page += 1
-	end
-
-	# reduce the service plans hash to only those plans that have active instances, and sort them
-	# in descenting order
-	service_plans.delete_if {| k, v | v['service_instance_count'] == 0 }
-	service_plans = service_plans.sort_by { |k, v| v['service_instance_count'].to_i }.reverse
+  page = 1
+  while page < 3
```

Comment:
any reason why we only look at the first two pages? not a blocker
(I know nothing changed but this was just shifting the indents)